numba/tests/test_function_type.py

```diff
@@ -593,9 +595,15 @@ def __init__(self, fname):
                 self._name = fname
                 if fname == 'cos':
                     addr = ctypes.cast(self.lib.cos, ctypes.c_voidp).value
-                    signature = float64(float64)
+                    if os.name == 'nt' and is32bits:
```

Comment:
Suggest using `from numba.core.config import IS_WIN32` for detecting windows.

Reply:
Fixed.